### PR TITLE
Add current target to longruns

### LINKED ISSUE #355: Update Longruns
## Purpose 

We need to revamp our longruns to reflect the current requirements and capabilities of ClimaAtmos. 

## To-do

- [x] adapt the `pipeline.yml`

## QA
- [x] show physically reasonable long runs
amip_paperplots
- [x] perf: 10h15m for n=64, compared to standalone ClimaAtmos 23h for 400d but with n=64 https://buildkite.com/clima/climaatmos-longruns/builds/1098#_ 
    - note: n64 v n32 not a huge speed up: https://buildkite.com/clima/climacoupler-longruns/builds/199 
    -

### PR DESCRIPTION
**Pull request by LenkaNovak**
Closes #355 

[ClimaCoupler-Longruns](https://buildkite.com/clima/climacoupler-longruns/builds/199)

Note: the old AMIP FINE benchmark crashes (unconverged SFs) after 120d. We will need to find the new stability region for these options, so it is commented for now. 

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

File: .buildkite/longruns/pipeline.yml [hunk 1]
```diff
@@ -124,10 +135,10 @@ steps:
           slurm_mem_per_cpu: 16G
           slurm_tasks_per_node: 2
 
-      - label: "Unthreaded AMIP FINE" # also reported by longruns with a flame graph
+      - label: "Unthreaded AMIP FINE" # also reported by longruns with a flame graph; 10d take 21h, so reducing to 1d
```
Comment (valeriabarra) [inline .buildkite/longruns/pipeline.yml:138]: Not sure if it is worth specifying "unthreaded" now that all runs are unthreaded

Comment (LenkaNovak) [reply .buildkite/longruns/pipeline.yml:138]: Good point. n=1 is more accurate here.

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -54,20 +54,31 @@ steps:
...
+      #     CLIMACORE_DISTRIBUTED: "MPI"
+      #     BUILD_HISTORY_HANDLE: ""
+      #   agents:
+      #     slurm_ntasks: 64
+      #     slurm_mem_per_cpu: 16G
+
+      - label: "MPI AMIP FINE: target longrun"
+        key: "amip_longrun_target"
+        command: "mpiexec julia --color=yes --project=experiments/AMIP/modular/ experiments/AMIP/modular/coupler_driver_modular.jl --run_name amip_longrun_target --coupled true  --anim true --surface_setup PrescribedSurface --dt_cpl 150 --energy_check false --mode_name amip --mono_surface false --vert_diff true --moist equil --rad clearsky --precip_model 0M --z_elem 35 --dz_bottom 50 --h_elem 12 --kappa_4 3e16 --rayleigh_sponge true --alpha_rayleigh_uh 0 --dt 150secs --t_end 140days --job_id amip_longrun2 --dt_save_to_sol 5days --dt_save_to_disk 1days --FLOAT_TYPE Float64"
+        artifact_paths: "experiments/AMIP/modular/output/amip/amip_longrun_target_artifacts/*"
```
Comment (akshaysridhar) [inline .buildkite/longruns/pipeline.yml:81]: LGTM thanks!

**Comment by valeriabarra** [comment]
FYI: When jobs fail with `Error: The command exited with status 137`, it is better to add memory, e.g. `slurm_mem: 20GB` than just retrying (which is time consuming)

**Comment by LenkaNovak** [comment]
Yeah, I was trying to avoid that but I think I will have to resort to that now, lol. Thanks for keeping an eye! 👀

**Pull request merged**